nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -356,7 +356,7 @@ in
         pkgs.media-player-info
       ];
 
-      services.xserver.displayManager.sddm = {
+      services.displayManager.sddm = {
```

Comment:
Similar changes should be done for Plasma 6 as well now that it's merged.

Reply:
There is already a PR for that

Reply:
I don't mean moving Plasma 6 itself, but replacing `services.xserver.displayManager.sddm` by `sevices.displayManager.sddm`.